component/src/main/java/me/androidbox/component/general/PhotoPicker.kt

```diff
@@ -40,6 +42,7 @@ import me.androidbox.component.ui.theme.photoTextColor
 fun PhotoPicker(
     listOfPhotoUri: List<String>,
     onPhotoUriSelected: (photoUri: Uri) -> Unit,
```

Comment:
Rememeber an event will have Local Photos, but also Remote Photos (in case it's an event that's on the backend)

And those, won't have a Uri. They will have a url

Reply:
Theoretically a URL is also a Uri and you could parse it into one, but I'd also keep them separate